src/serde/de_br.rs

```diff
@@ -83,6 +129,86 @@ pub fn node_from_bytes_backrefs_record(
     Ok((ret, backrefs))
 }
 
+pub fn traverse_path_with_vec(
```

Comment:
this is a pretty big function for not having a unit test.

Reply:
Added a unit test